src/Components/Navbar.js

```diff
@@ -12,15 +13,19 @@ function Navbar() {
 
     const handleTotal = () => {
         try {
-            const cartItems = JSON.parse(localStorage.getItem('cart')) || []
-            setCartItems(cartItems)
+            const cart = JSON.parse(localStorage.getItem('cart')) || [];
+            setCartItems(cart);
         } catch (error) {
-            console.error(error);
+            console.error('Error parsing cart data:', error);
         }
     };
-    const totalAmount = cartItems.reduce((sum, item) => sum + item.quantity, 0)
+
+    const totalAmount = cartItems.reduce((sum, item) => sum + item.quantity, 0);
+
     useEffect(() => {
         handleTotal();
+        const intervalId = setInterval(handleTotal, 100);
```

Comment:
why did you add this?

Reply:
To handle the cart count

Reply:
No need to add setInterval. Please remove this and handle without this

Reply:
okay